src/coreclr/src/inc/pinvokeoverride.h

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+/*****************************************************************************
+ **                                                                         **
+ ** pinvokeoverride.h - PInvoke binding override                            **
+ **                                                                         **
+ *****************************************************************************/
+
+#ifndef _PINVOKEOVERRIDE_H_
+#define _PINVOKEOVERRIDE_H_
+
+#ifndef _MSC_VER
+#define __stdcall
+#endif
+
+typedef const void* (__stdcall PInvokeOverrideFn)(const char* libraryName, const char* entrypointName);
+
+class PInvokeOverride
+{
+private:
+    static PInvokeOverrideFn* s_overrideImpl;
+
+public:
+    static void SetPInvokeOverride(PInvokeOverrideFn* overrideImpl);
+    static const void* TryGetMethodImpl(const char* libraryName, const char* entrypointName);
+
+private:
+    static const void* DefaultResolveDllImport(const char* libraryName, const char* entrypointName);
+};
+
+#endif // _PINVOKEOVERRIDE_H_
+// EOF =======================================================================
```

Comment:
This isn't a recent style in the coreclr code base. I don't think we should try and continue it.

Reply:
Agree. Can you please delete it from the 3 files that are using this while you are on it?

Reply:
This was copied from Bundle.h   
Sometimes it is hard to tell whether some peculiar formatting or commenting is expected style for a particular area.